ethsigner/commandline/src/main/java/tech/pegasys/ethsigner/EthSignerBaseCommand.java

```diff
@@ -107,6 +107,15 @@
       arity = "1")
   private String downstreamHttpHost = InetAddress.getLoopbackAddress().getHostAddress();
 
+  @SuppressWarnings("FieldMayBeFinal")
+  @Option(
+      names = {"--downstream-http-request-args"},
+      description =
+          "Query String to append to the http request (e.g. full infura url with project id)",
```

Comment:
While the PR makes ethsigner usable by infura - this change would work for anyone who uses path-based routing - thus, it'd be good to keep the hep text agnostic (Rather than infura specific)

Reply:
Updated the description to be more inline with a more generic functionality not specific to infura.